apiserver/facades/schema.json

```diff
@@ -20758,6 +20758,9 @@
         "Description": "KeyManagerAPI provides api endpoints for manipulating ssh keys",
         "Version": 1,
         "AvailableTo": [
+            "controller-machine-agent",
+            "machine-agent",
+            "unit-agent",
```

Comment:
Was this intentional?

Reply:
This need to be fixed before landing. It indicates an issue with the tool to generate the schema json.

Reply:
Yep schema gen is broken at the moment. We know about it and a plan is in place to fix it.

Reply:
Yeah, this is just annoying. It's todo with how things are constructed in the building of the schema. I've got a proof of concept to fix it, but it's a lot of work. Something I don't want to take on atm.

Reply:
This is the fix, but you need to do it to all facades. https://github.com/juju/juju/pull/15822/files

Reply:
This should be followed up. I will forward the relevant email.